src/main/resources/view/DarkTheme.css

```diff
@@ -176,14 +176,15 @@
 
 .vbox_background {
     -fx-background-color: #3e7b91;
-    -fx-padding: 5 5 5 5;
+    -fx-padding: 10 10 10 10;
```

Comment:
if everything is 10 just put a single number 10. lel

Reply:
Omg I didn't know that you could do this, thanks for the tip!